suseconnect/suseconnect.go

```diff
@@ -186,7 +186,7 @@ func connectMain() {
 	// filesystem can handle operations from SUSEConnect for specific actions
 	// which require filesystem to be read write (aka writing outside of /etc)
 	// /etc is writable at any time, system token roation works just fine
-	if deRegister || rollback || cleanup {
+	if deRegister || cleanup {
```

Comment:
It would be great to align the rules here with the comment above and https://confluence.suse.com/display/scc/SLE+Micro. 
It's not clear why de-register and cleanup need r/w access outside of /etc.

Reply:
I haven't checked the code, but I was also wondering whether these rules are necessary at all... At least they didn't seem critical because the commands are not used in MicroOS' default scripts, so I didn't mention them.